### DISCUSSION TITLE
Sending Messages

### DISCUSSION THREAD

**Discussion by BitForger** [slack-go/slack#1005] [Q&A] [answered]
Hey!

I'm trying to use this library to send a message but I'm not sure how I'm supposed to do that. I have done something similar to this
```go
api := slack.New()
msg := slack.Message{}

slack.addBlockMessage(msg, slack.TextBlockObject{
    Type: slack.MarkdownType,
    Text: "Some text",
})
```

but I'm not sure how to send that message.

**Comment by kanata2** [comment, marked as answer]
See examples or official API doc.

- https://github.com/slack-go/slack/blob/master/examples/messages/messages.go
- https://api.slack.com/methods/chat.postMessage
